released/packages/coq-ssprove/coq-ssprove.0.2.0/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name: "coq-ssprove"
+version: "0.2.0"
+synopsis: "A Foundational Framework for Modular Cryptographic Proofs"
+maintainer: "philipp@haselwarter.org"
+authors: ["SSProve team"]
+homepage: "https://github.com/SSProve/ssprove"
+bug-reports: "https://github.com/SSProve/ssprove/issues"
+dev-repo: "git+https://github.com/SSProve/ssprove.git"
+license: "MIT"
+depends: [
+  "coq" {(>= "8.18" & < "8.20~")}
+  "coq-equations" {(>= "1.3+8.18")}
+  "coq-mathcomp-ssreflect" {(>= "2.1.0")}
+  "coq-mathcomp-analysis" {>= "1.0.0"}
+  "coq-extructures" {(>= "0.4.0" & < "dev")}
+  "coq-deriving" {(>= "0.2.0" & < "dev")}
+]
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+tags: [
+  "keyword:cryptography"
+  "logpath:Mon"
+  "logpath:Relational"
+  "logpath:Crypt"
```

Comment:
@4ever2 @spitters just flagging up that these are very confusing logpaths for a project called SSProve. I highly recommend setting up logpaths in the style of `SSProve.Relational`, `SSProve.Crypt`, `SSProve.Mon`, otherwise we risk confusion and clashes with other packages.